src/EditorFeatures/Core/Implementation/InfoBar/EditorInfoBarService.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Extensions;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Internal.Log;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Workspaces
+{
+    [ExportWorkspaceService(typeof(IInfoBarService))]
```

Comment:
I'm surprised there was no error on this.  I think you need to add ", Shared" to the attribute, and suck in "using System.Composition".